### PR TITLE
feat:add card component

### PR DESCRIPTION
**Pull request by rahulmistry751**
Added card component
- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Horizontal cards
card

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

**Review by JowelTisso — APPROVED** [review]

File: components/css/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    display:none;
+}
+
+/* Horizontal cards */
+.card-hori{
+    width:30rem;
+    height:auto;
+    flex-direction:row;
+    flex-wrap:wrap;
+    /* justify-content: space-evenly; */
```
Comment (pavan6115) [inline components/css/cards/cards.css]: Remove unwanted comments after final edits done to the code, makes code look more clean

**Review by pavan6115 — APPROVED** [review]

**Comment by pavan6115** [comment]
In the title of PR write something like - PR : Card Component
You are writing commit message type in pr title. Don't do that.

**Comment by rahulmistry751** [comment]
> In the title of PR write something like - PR : Card Component You are writing commit message type in pr title. Don't do that.

Ok. Thanks

File: components/css/cards/cards.css (hunk 1 above)
Comment (rahulmistry751) [reply components/css/cards/cards.css]: Yes

**Pull request merged**